src/Controller/ArticlesController.php

```diff
@@ -696,6 +696,19 @@ public function risAction(Request $request, string $id) : Response
         return new Response(preg_replace('~\R~u', "\r\n", $this->get('templating')->render('::article.ris.twig', $arguments)), Response::HTTP_OK, ['Content-Type' => 'application/x-research-info-systems']);
     }
 
+    public function xmlAction(Request $request, string $id, int $version = null) : Response
+    {
+        $arguments = $this->defaultArticleArguments($request, $id, $version);
+
+        $xml = $arguments['item']
+            ->then(Callback::method('getXML'))
```

Comment:
Methods are case insensitive, but this is called `getXml` in the original I think: https://github.com/elifesciences/api-sdk-php/blob/df953302a4931eed85166a55649c0ad9f436d526/src/Model/ArticleVersion.php#L221 I think